pkg/clientgocred/cred.go

```diff
@@ -0,0 +1,370 @@
+// package clientgocred an ExecCredential object prints to stdout . This
+// is defined by Client-go Credential: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins
+// This library can be used with GKE Clusters for use with kubectl and custom
+// k8s clients.
+package clientgocred
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Sounds good, I ll take the change. But, the reason to name it meta instead of metav1, is that the version can be changed without any changes to  the name.

Reply:
In general, you can't change versions without reworking the code.